docs/release-notes.md

```diff
@@ -94,7 +98,7 @@ key](https://getfedora.org/security/).
 ### Docs changes
 
 - Document consequence of setting `systemd.units.mask` to false
-- Document `grub` section _(openshift 4.15.0-exp)_
+- Document `grub` section _(openshift 4.18.0-exp)_
```

Comment:
Maybe I did this in the past, but let's try not changing past release notes.

We can add a note for the current release instead.

Someting like:
```
Support and documentation for `grub` section moved to OpenShift 4.18.0-experimental spec.
```

Reply:
Ah, ok, makes sense. will not do in the future; thank you for fixing the nits!